src/modules/qt/common.cpp

```diff
@@ -63,13 +63,24 @@ bool createQApplicationIfNeeded(mlt_service service)
     return true;
 }
 
-void convert_qimage_to_mlt_rgba(QImage *qImg, uint8_t *mImg, int width, int height)
+mlt_image_format choose_image_format(mlt_image_format format)
+{
+#if QT_VERSION >= QT_VERSION_CHECK(5, 12, 0)
+    if (format == mlt_image_rgba64) {
+        return mlt_image_rgba64;
+    }
+#endif
+    return mlt_image_rgba;
+}
+
+void convert_qimage_to_mlt(QImage *qImg, uint8_t *mImg, int width, int height)
 {
 #if QT_VERSION >= QT_VERSION_CHECK(5, 2, 0)
     // QImage::Format_RGBA8888 was added in Qt5.2
-    // Nothing to do in this case  because the image was modified directly.
+    // QImage::Format_RGBA64 was added in Qt5.12
```

Comment:
We do not need to keep the old code for Qt 5.2. No one uses that anymore. Minimum Qt 5 version we support now is its last, which is also LTS: 5.15